runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -36,12 +36,12 @@ use super::{
     assert_last_event, channel_bag_witness, clone_curator_group,
     create_data_object_candidates_helper, generate_channel_creation_params, insert_content_leader,
     insert_curator, insert_distribution_leader, insert_storage_leader, max_curators_per_group,
-    member_funded_account, setup_video, setup_worst_case_curator_group_with_curators,
```

Comment:
why not simply the `use *` version?

Reply:
Addressed in https://github.com/Joystream/joystream/pull/4221/commits/c625d8a09b40545ffb5d8259c75d8571249545b8